projectroles/tests/test_templatetags.py

```diff
@@ -618,3 +618,211 @@ def test_get_sidebar_app_legend(self):
         self.assertEqual(
             tags.get_sidebar_app_legend('Update Project'), 'Update<br />Project'
         )
+
+    def test_get_sidebar_links_home_view(self):
+        """Test get_sidebar_links() on the home view"""
+        url = reverse('home')
+        req_factory = RequestFactory()
+        with self.login(self.user):
+            request = req_factory.get(url)
+            request.resolver_match = resolve(url)
+            request.user = self.user
+            self.assertEqual(
+                tags.get_sidebar_links(request),
+                [],
+            )
+
+    def test_get_sidebar_links_project_view(self):
```

Comment:
This is specifically for the project detail view, so `_project_detail` would be the appropriate name. Same for the docstring. Also note that we want to make unit test docstrings as concise as possible, so omitting e.g. articles is fine.